src/realm/utilities.cpp

```diff
@@ -307,6 +307,24 @@ void process_mem_usage(double& vm_usage, double& resident_set)
 }
 #endif
 
+#ifdef _WIN32
+int gettimeofday(struct timeval * tp, struct timezone * tzp)
```

Comment:
It's to mimic the posix version of the method. Could be handy that they have the same prototype to avoid #ifdef for future use (in current use the return value is ignored though)